geometry_msgs/Point position
hit_spider/hexapod_RPY orientation